tools/format_results.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python
+import argparse
+import pandas as pd
+from statistics import multimode
+import statistics
+
+print('Formating')
+parser = argparse.ArgumentParser()
+parser.add_argument('-i_dir', '--input_dir',required = False, help="input dir (ex. pred-alchemist)",  type=str)
+parser.add_argument('-c','--cancer', required=True, help='<Required> Set flag, where each cancer is listed', type=str)
+parser.add_argument('-m','--method_list', required=True, help='<Required> Set flag, where each method is listed', type=str)
+parser.add_argument('-o', '--outfile',required = True, help="output file name",  type=str)
+args = parser.parse_args()
+
+# combine all 5 methods together
+results_d = {}
+for m in args.method_list.split('.'):
+    results_d[m] = pd.read_csv('{}/midway/{}_{}_format2.tsv'.format(args.input_dir, m, args.cancer), sep='\t', index_col=0)
+methods = list(results_d.keys())
+first = 1
+for method,df in results_d.items():
+    if first == 1:
+        base = df
+        first = 0
+    else:
+        base = pd.merge(base, df, on=['sampleID', 'TCGA_cohort', 'platform'], how='outer')
+
+# all absolute sample subtype call column
+# this shows the most voted for subtype
```

Comment:
voted-for

Reply:
updated in commit [9b09147](https://github.com/NCICCGPO/gdan-tmp-models/pull/54/commits/9b09147d35ceab6a336723b6c74ef06ac86c0ce2)